crates/sui-framework/sources/governance/validator.move

```diff
@@ -53,8 +53,13 @@ module sui::validator {
     /// Invalidworker_address field in ValidatorMetadata
     const EMetadataInvalidWorkerAddr: u64 = 7;
 
+    /// Commission rate set by the validator is higher than the threshold
+    const ECommissionRateTooHigh: u64 = 8;
+
     const EInvalidProofOfPossession: u64 = 0;
 
+    const MaxCommissionRate: u64 = 1_000_000; // Max rate is 100%, which is 1M base points
```

Comment:
I'm pretty sure 1% is 100 basis points so 100% should be 10000 basis points.

Reply:
Ouch, right. Max commission rate of 10000% is too much